Dockerfile

```diff
@@ -9,6 +9,12 @@ ENV PATH /root/.local/bin:$PATH
 RUN mkdir -p /app/user
 WORKDIR /app/user
 
+# Add a /app/GIT_HEAD_REF file with the git commit SHA that is currently
+# deployed
+COPY .git .git
```

Comment:
Yep, fixed in 9237fcd86ecedb62633ec5b1c87efb1a117af547 and confirmed to work with Croniker.